command.go

```diff
@@ -121,6 +124,15 @@ func UnRegister(topic string, channel string) *Command {
 	return &Command{[]byte("UNREGISTER"), params, nil}
 }
 
+// SyncState creates a new Command to sync a topic/channel pause state changes.
+func SyncState(topic string, channel string, state int) *Command {
```

Comment:
This command should probably send a JSON-serialized dictionary of key/values, e.g.

```json
{
    "paused": true
}
```

Reply:
Yeah, I think it's all about extendability, by Json, we can use multiple different key-value pairs to represent different states.
What I thought at the time was that we can use integer value to represent different states, like 0 for unpaused, 1 for paused, 2,3,4...for else.
Maybe Json extendability is better? and also readability.

Reply:
agree change to json.  : )